bin/cylc-get-host-metrics

```diff
@@ -35,19 +35,18 @@ keys as requested via the OPTIONS:
 If no options are specified, --load and --memory are invoked by default.
 """
 
-import sys
-if '--use-ssh' in sys.argv[1:]:
-    sys.argv.remove('--use-ssh')
-    from cylc.remote import remrun
-    if remrun():
-        sys.exit(0)
-
 import os
 import re
 from subprocess import Popen, PIPE, CalledProcessError
 import json
 
 from cylc.option_parsers import OptionParser
+import sys
+if '--use-ssh' in sys.argv[1:]:
+    sys.argv.remove('--use-ssh')
+    from cylc.remote import remrun
+    if remrun():
+        sys.exit(0)
 
```

Comment:
As per a previous discussion (can't recall exactly when or where) can this change be reverted please?  We use `cylc.remote.remrun()` in most commands before importing modules that will never be used if the command just re-invokes itself on another host. (Efficiency trumps "imports not at top of module" warnings; or if it doesn't, we should be consistent and change this in all the commands).

Reply:
The  [pep8] module level import not at top of file [E402] causes a codacy build error. I can revert and add a # noqa: E402

Reply:
BTW Codacy "fails" a PR if the number of issues goes up, but that doesn't matter so long as we agree that the "new" issues are not vaild.  Over time we want to deal with or ignore (like #nosec) all issues, but it's OK to ignore Codacy in cases like this.